superset-frontend/src/SqlLab/components/RunQueryActionButton/index.tsx

```diff
@@ -27,6 +27,7 @@ import {
   DropdownButtonProps,
 } from 'src/components/DropdownButton';
 import { detectOS } from 'src/utils/common';
+import { useMemo } from '@storybook/addons';
```

Comment:
This one should be imported from React

Reply:
Correct... and we can add this (and the lodash one) as an eslint restricted-imports rule.